pkg/gocui/edit.go

```diff
@@ -33,7 +33,7 @@ var DefaultEditor Editor = EditorFunc(simpleEditor)
 // simpleEditor is used as the default gocui editor.
 func simpleEditor(v *View, key tcell.Key, ch rune, mod tcell.ModMask) {
 	switch {
-	case key == tcell.KeyRune && ch != 0 && mod == 0:
+	case key == tcell.KeyRune && ch != 0 && mod <= tcell.ModShift:
```

Comment:
Using <= seems like a non-obvious way to allow "none-or-shift" (relies on mod ordering). Would prefer `(mod==ModNone || mod==ModShift)`

Reply:
thx. I've fixed it!